atramhasis/views/views.py

```diff
@@ -252,13 +245,17 @@ def results_csv(self):
             'filename': 'atramhasis_export'
         }
 
+    @view_config(route_name='scheme_tree_html', renderer='scheme_tree.jinja2')
     @view_config(route_name='scheme_tree', renderer='json', accept='application/json')
     def results_tree_json(self):
         scheme_id = self.request.matchdict['scheme_id']
         locale = self.request.locale_name
         dicts = self.get_results_tree(scheme_id, locale)
         if dicts:
-            return dicts
+            if 'application/json' in self.request.accept:
```

Comment:
Hierdoor zal ook het browserl geval de json terugsturen naar de jinja pagina.
`application/json` in `*/*` is `True`.

Reply:
Ik heb nu gecheckt dat 'text/html' niet in de Accept header zit.

De enige manier om nu json te krijgen is door het expliciet mee te geven.